src/imap/idle.rs

```diff
@@ -65,68 +64,43 @@ impl Imap {
                 Interrupt(InterruptInfo),
             }
 
-            if self.skip_next_idle_wait {
```

Comment:
Ok, seems `skip_next_idle_wait` was never set to `true`.

Reply:
yep, so only dead code at the moment